src/components/Transition/Fade.js

```diff
@@ -4,42 +4,47 @@ import { Transition } from 'react-transition-group';
 import anime from 'animejs';
 import { getCSSVariableAsObject, getCSSVariableAsNumber } from '../../utils/CSSVariables';
 
-const duration = {
+const durationFast = {
   enter: getCSSVariableAsNumber('--animation-duration-fast-ms'),
   exit: getCSSVariableAsNumber('--animation-duration-fast-ms'),
 };
 
-const enterAnimation = {
+const durationSlow = {
+  enter: getCSSVariableAsNumber('--animation-duration-slow-ms'),
+  exit: getCSSVariableAsNumber('--animation-duration-slow-ms'),
+};
+
+const enterAnimation = duration => ({
   opacity: [0, 1],
   elasticity: 0,
   easing: getCSSVariableAsObject('--animation-easing-js'),
   duration: duration.enter,
-};
+});
 
-const exitAnimation = {
+const exitAnimation = duration => ({
   opacity: [1, 0],
   elasticity: 0,
   easing: getCSSVariableAsObject('--animation-easing-js'),
   duration: duration.exit,
-};
+});
 
-const Fade = ({ children, ...props }) => (
+const Fade = ({ children, isSlow, ...props }) => (
```

Comment:
What do you think about implementing this more generally?

e.g.

```jsx
const Fade = ({ children, duration, ...props }) => (
  <Transition
    {...props}
    timeout={duration}
    onEnter={
      (el) => {
        anime({
          targets: el,
          ...enterAnimation(duration),
        });
      }
    }
    onExit={
      (el) => {
        anime({
          targets: el,
          ...exitAnimation(duration),
        });
      }
    }
    appear
    unmountOnExit
  >
    { children }
  </Transition>
);

Fade.propTypes = {
  children: PropTypes.any.isRequired,
  duration: PropTypes.object,
};

Fade.defaultProps = {
  children: null,
  duration: defaultDuration,
};
```

Reply:
Makes sense to me!